syntax = "proto3";

package ledgerd;

service LedgerdService {
    rpc Ping(PingRequest) returns (PingResponse) {}
}

message PingRequest {
    string ping = 1;
}
message PingResponse {
    string pong = 1;
}